pkg/backup/backup/backup_cleaner.go

```diff
@@ -127,7 +127,8 @@ func (bc *backupCleaner) makeCleanJob(backup *v1alpha1.Backup) (*batchv1.Job, st
 
 	podSpec := &corev1.PodTemplateSpec{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels: backupLabel.Labels(),
+			Labels:      backupLabel.Labels(),
+			Annotations: backup.Annotations,
```

Comment:
Should also change for serviceAccount case, right?

Reply:
done